src/julia-parser.scm

```diff
@@ -1,4 +1,4 @@
-;; Operator precedence table, lowest at top
+; Operator precedence table, lowest at top
```

Comment:
By scheme coding standards `;;` is correct here; the rule is that `;;` is for comments starting in column 1 and comments with `;` get indented. We don't follow this perfectly but it's not really worth changing.

Reply:
just noticed that, must have been an accident in my editor